js/audio/interactive/InteractiveAudio.svelte

```diff
@@ -202,7 +202,7 @@
 		recording = false;
 
 		if (streaming) {
-			dispatch("stop_recording");
+			dispatch("stop_recording", value);
```

Comment:
why does the stop_recording event send data? I don't think we make this accessible to users anywhere